ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionBuilderModule.java

```diff
@@ -378,7 +378,7 @@ private SafeFuture<BuilderPayload> getPayloadFromBuilder(
             () ->
                 new RuntimeException(
                     "Unable to get payload from builder: builder endpoint not available"))
-        .getPayload(signedBlindedBlockContainer)
+        .getPayload(signedBlindedBlockContainer.getSignedBlock())
```

Comment:
As I understand we will need start from block higher, you are just cutting changes here for the moment. I'd make at least input of this function with Block, and update logging line, just at least to make it correct

Reply:
good catch, I like it.